homeassistant/components/upnp/config_flow.py

```diff
@@ -82,6 +87,9 @@ async def async_step_user(self, user_input: Optional[Mapping] = None):
                         for discovery in self._discoveries
                     }
                 ),
+                vol.Optional(
+                    CONF_SCAN_INTERVAL, default=DEFAULT_SCAN_INTERVAL,
```

Comment:
We normally don't allow optional items on the main config flow. I think we should move this from the main flow into an options flow.

Reply:
See #35474